code_examples/sdk_examples/src/core_features/claiming/05_verify_presentation.ts

```diff
@@ -2,8 +2,30 @@ import * as Kilt from '@kiltprotocol/sdk-js'
 
 export async function verifyPresentation(
   presentation: Kilt.ICredentialPresentation,
-  challenge: string | undefined = undefined
+  {
+    challenge,
+    trustedAttesterUris = []
+  }: {
+    challenge?: string
+    trustedAttesterUris?: Kilt.DidUri[]
+  } = {}
 ): Promise<void> {
   // Verify the presentation with the provided challenge.
   await Kilt.Credential.verifyPresentation(presentation, { challenge })
+
+  // Verify the credential attestation by checking on the blockchain.
+  const api = Kilt.ConfigService.get('api')
+  const attestationChain = await api.query.attestation.attestations(
+    presentation.rootHash
+  )
+  const attestation = Kilt.Attestation.fromChain(
+    attestationChain,
+    presentation.rootHash
+  )
+  if (attestation.revoked) {
+    throw "Credential has been revoked and hence it's not valid."
+  }
+  if (!trustedAttesterUris.includes(attestation.owner)) {
+    throw `Credential was issued by ${attestation.owner} which is not in the provided list of trusted attesters: ${trustedAttesterUris}.`
```

Comment:
Can we throw an `new Error` and also the quotes should be single quotes

Reply:
It's an automatic linter fix, which I guess switches to double quotes when the string contains a `'`.

Reply:
Ah never mind than.

Reply:
https://github.com/KILTprotocol/docs/pull/201/commits/501b348790fd59c790f0e72432f491e6e1b1c2d2